src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Commands/KillProcessCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.WindowWalker.Components;
+using Microsoft.CmdPal.Ext.WindowWalker.Helpers;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Commands;
+
+internal sealed partial class KillProcessCommand : InvokableCommand
+{
+    private readonly Window _window;
+
+    public KillProcessCommand(Window window)
+    {
+        Icon = new("\xE74D"); // Delete symbol
+        Name = $"{Resources.wox_plugin_windowwalker_Kill} (Ctrl+Delete)";
```

Comment:
Get rid of the `ctrl+delete`. We can stick it on the context menu item as a `RequestedShortcut` if you want, but putting the text here seems goofy

Reply:
Valid. Copied and pasted from Run plugin - can axe those for sure.